libs/nestjs-admin/src/utils/pagination.ts

```diff
@@ -0,0 +1,99 @@
+import lodash = require('lodash')
+
+export function getPaginationRanges(current: number, resultsPerPage: number, totalResults: number) {
+  const delta = 2
+  const pages = Math.ceil(totalResults / resultsPerPage)
+  current = Math.min(Math.max(1, current), pages)
+
+  const nranges = [
+    [1],
+    lodash.range(Math.max(1, current - delta), Math.min(current + delta + 1, pages)),
+    [pages],
+  ]
+
+  return nranges.reduce(
+    function(ranges, currRange) {
+      if (currRange.length === 0) return [...ranges]
+
+      const prevRange: number[] = ranges[ranges.length - 1]
```

Comment:
```suggestion
      const prevRange: number[] = ranges.slice(-1)
```

the canonical way to get the last element